components/zerotier/zerotier.app.mjs

```diff
@@ -32,6 +32,48 @@ export default {
         }));
       },
     },
+    memberHidden: {
+      type: "boolean",
+      label: "Member Hidden?",
+      description: "Whether or not the member is hidden in the UI",
+      optional: true,
+      default: null,
+    },
+    memberName: {
+      type: "string",
+      label: "Member Name",
+      description: "User defined name of the member",
+      optional: true,
+      default: null,
```

Comment:
Since `memberName` is a string, the default value should be a string. I think in this case, it would be best not to include a default value.
```suggestion
```